src/carma-streets/parse_kafka_logs.py

```diff
@@ -124,7 +124,40 @@ def parse_timesync_to_csv(inputfile: Path, outputfile: Path):
             print('Finished writing all entries successfully')
         else:
             print(f'WARNING: Skipped {skipped_messages} due to errors. Please inspect logs')
-   
+
+def parse_detectedobject_to_csv(inputfile: Path, outputfile: Path):
+    """Function to parse timesync Kafka Topic log file and generate csv data of all time sync messages
+
+    Args:
+        inputfile (Path): Path to Kafka Topic log file
+        outputfile (Path): File name (excluding file extension) of desired csv file
+    """
+    intersection_lidar_x_offset = 263.799988
+    intersection_lidar_y_offset = 178.050003
+    intersection_lidar_z_offset = 0.0
```

Comment:
We should avoid hard coding these values.

Reply:
where do you suggest I put them?